protocols/xdg-output-unstable-v1.xml

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please don't embed the protocol in Sway, use the XML file from wayland-protocols instead.